### ISSUE TITLE
Loss Weighting Overridden in IBP Training

### ISSUE THREAD

**Issue by GiulioZizzo** [Trusted-AI/adversarial-robustness-toolbox#2102]
**Describe the bug**
If the user chooses not to use a scheduler for the loss weighting, and wants to use a fixed weighting, their supplied value is overridden and a default one is used instead.

**To Reproduce**
1. Go to ibp_certified_trainer_pytorch.py
2. Scroll down to line 304
4. loss_weighting_k = 0.1 should be loss_weighting_k = self.loss_weighting

**System information (please complete the following information):**
 - OS: Mac OS
 - Python version: Python 3.8
 - ART version or commit number: 1.14.0
 - PyTorch 1.13.1
